coworking-opportunities.qmd

```diff
@@ -0,0 +1,38 @@
+---
+title: "Coworking Opportunities"
+---
+
+Coworking is one of the most actionable, low cost, immediate things groups can do to support each other and build new skills and workflows. Here are some opportunities to work and learn alongside NOAA Fisheries colleagues and others in the Openscapes community.
```

Comment:
```suggestion
Coworking is one of the most actionable, immediate things groups can do to support each other and build new skills and workflows. Here are some opportunities to work and learn alongside NOAA Fisheries colleagues and others in the Openscapes community.
```

Reply:
Removing "low cost" here since for the CNG audience that was intended to say "it's not free, it's someone's time to intentionally lead", but here we are doing that work and inviting so I think it's better to leave off